src/lib/analytics/AnalyticsClass.js

```diff
@@ -1,30 +1,59 @@
 // @flow
-import { assign, debounce, forIn, get, isEmpty, isNumber, isString, pick, remove, toLower, values } from 'lodash'
+import {
+  assign,
+  debounce,
+  forIn,
+  get,
+  isEmpty,
+  isNumber,
+  isString,
+  isUndefined,
+  negate,
+  once,
+  pick,
+  pickBy,
+  remove,
+  toLower,
+  values,
+} from 'lodash'
+
 import { isMobileReactNative } from '../utils/platform'
 import { LogEvent } from '../logger/pino-logger'
 import { ExceptionCategory } from '../logger/exceptions'
 import { ANALYTICS_EVENT, ERROR_LOG } from './constants'
 
-const savedErrorMessages = new WeakMap()
-
 export class AnalyticsClass {
   apis = {}
 
-  constructor(apis, rootApi, Config, loggerApi) {
+  savedErrorMessages = new WeakMap()
```

Comment:
what is this?

Reply:
it was outside class, i just moved it inside